test/functional/rpc_rawtransaction.py

```diff
@@ -14,6 +14,7 @@
 
 from test_framework.test_framework import BitcoinTestFramework
 from test_framework.util import *
+from test_framework.mininode import CTransaction, ToHex
```

Comment:
`CTransaction` is imported but unused
`ToHex` is defined in test_framwork.messages and should be imported from there.

Please sort imports alphabetically.

Reply:
Thanks for the feedback @jnewbery! I have addressed your feedback in 815299e and sorted the imports. Please note that `CTransaction` is used.

Reply:
Thanks for the feedback @jnewbery! I have addressed your feedback in 815299e and sorted the imports. Please note that CTransaction is used.